src/components/problem/styledProblem.js

```diff
@@ -4,23 +4,58 @@ import PercentImg from '../../images/plastic-percents.svg'
 export const ProblemWrapper = styled.section`
   padding: 120px 0;
   background: #1C383C;
+
+  @media( max-width: 1023px ) {
+    padding: 80px 0;
+  }
 `;
 
 export const Grid = styled.div`
-  width: 1160px;
+  width: 80.56%;      // 1160 / 1440
   margin: 0 auto;
   display: grid;
   grid-template-columns: repeat(3, 1fr);
   grid-template-rows: 348px 369px;
   gap: 80px 60px;
+
+  @media( max-width: 1280px ) {
+    width: 95.9%;      // 982 / 1024
+    gap: 80px 20px;
+  }
+
+  @media( max-width: 1023px ) {
+    width: 94.7%;      // 727 / 768
+    grid-template-rows: 226px 229px 300px;
+    gap: 40px 20px;
+  }
+  @media( max-width: 767px ) {
+    width: 90%;       // 288 / 320
+    display: flex;
+    flex-direction: column;
+    gap: 40px;
+  }
 `;
 
 export const Cell = styled.div`
+  @media( max-width: 1023px ) {
+    grid-row: 3/4;
+  }
+  @media( max-width: 767px ) {
+    order: 3;
+    margin: 0 auto;
+    width: 229px;
+  }
 `;
 
 export const TitleWrapper = styled.div`
   grid-column: 2/4;
   margin-top: 56px;
+
+  @media( max-width: 1023px ) {
```

Comment:
According to the figma design, this is slightly smaller than 2 columns width in the 1024 px design

Reply:
Nice. I got it covered now.

Reply:
It's not perfect, but it's really damn close.  I think this is fine.

Reply:
haha yeah i eye-balled this one. good enough